include/zephyr/drivers/sensor/bmi270.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2023 Bosch Sensortec GmbH. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Extended public API for Bosch BMI270 sensor
+ *
+ * This exposes an API to the sensor's CRT feature which
+ * is specific to the application/environment and cannot
+ * be expressed within the sensor driver abstraction.
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_SENSOR_BMI270_H_
+#define ZEPHYR_INCLUDE_DRIVERS_SENSOR_BMI270_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <zephyr/drivers/sensor.h>
+
+/* Sensor specific attributes */
+enum sensor_attribute_bmi270 {
+	/* Programs the non volatile memory(nvm) of bmi270 */
+	BMI270_SENSOR_ATTR_NVM_PROG = SENSOR_ATTR_PRIV_START,
+	/* Set BMI270 user gain compensation */
+	BMI270_SENSOR_ATTR_GAIN_COMP
+};
+
+/* Macros for setting gyro gain compensation */
+#define BMI270_GYR_GAIN_EN                 0x01
+#define BMI270_GYR_GAIN_DIS                0x00
+
+/* Helper Macros for setting gyro gain compensation */
+#define SENSOR_VALUE_WITH_GAIN(var_name, gain_value) \
```

Comment:
These seem to be BMI270 specific helper macros without any prefix to them which might lead to some confusion